src/widget.js

```diff
@@ -1,23 +1,140 @@
-var buttonUrl = "https://superhero.com/buttons/v1/";
-document.addEventListener("DOMContentLoaded", function(event) {
-  var buttons = document.querySelectorAll("a.superhero-tip-button");
-  var iframes = [];
-  for (var i = 0; i < buttons.length; i++) {
-    var size = buttons[i].getAttribute("data-size") ? buttons[i].getAttribute("data-size") : "icon";
-    var href = buttons[i].getAttribute("data-href") ? buttons[i].getAttribute("data-href") : window.location.href;
-    var account = buttons[i].getAttribute("data-account");
-
-    var iframe = document.createElement('iframe');
-    iframe.src = buttonUrl + size + "?url=" + href;
-    iframe.setAttribute('style','max-width:210px;max-height:120px;overflow:hidden;');
-    iframe.setAttribute("scrolling", "no");
-    iframe.setAttribute("frameborder", "0");
-    iframe.className = "superhero-tip-button";
-    if (account) iframe.setAttribute("data-account", account);
-    iframes[i] = iframe;
+const { detect } = require('detect-browser');
+
+const defaults = {
+  size: 'icon',
+  url: window.location.href,
+  inlineCss: false,
+  version: 1,
+  all: false
+};
+
+const templates = {
+  icon: `<div class="superhero-button-link-holder">
+            <a id="superhero-tip-link" class="superhero-button-link" target="_blank">
```

Comment:
Would be better to avoid IDs because multiple buttons may be on the same page
```suggestion
            <a class="superhero-tip-link superhero-button-link" target="_blank">
```